.env

```diff
@@ -5,29 +5,45 @@ REACT_APP_PINAKION_KOVAN_ADDRESS=0x1ee318dbc19267dbce08f54a66ab198f73ede356
 REACT_APP_POLICY_REGISTRY_KOVAN_ADDRESS=0xfc53d1d6ddc2c6cdd403cb7dbf0f26140d82e12d
 REACT_APP_UNISWAP_V2_ROUTER_02_KOVAN_ADDRESS=0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D
 REACT_APP_UNISWAP_V2_FACTORY_KOVAN_ADDRESS=0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f
+REACT_APP_TOKEN_BRIDGE_KOVAN_ADDRESS=0xA960d095470f7509955d5402e36d9DB984B5C8E2
+REACT_APP_WEB3_FALLBACK_KOVAN_URL=https://kovan.infura.io/v3/f1c9535b961648f7bd18209b00e11163
+
+REACT_APP_KLEROS_ADDRESS=0x211f01e59b425253c0a0e9a7bf612605b42ce82c
+REACT_APP_KLEROS_LIQUID_ADDRESS=0x988b3a538b618c7a603e1c11ab82cd16dbe28069
+REACT_APP_KLEROS_LIQUID_EXTRA_VIEWS_ADDRESS=0xda47f3252bb03c5c7950d7bb2fd32637fc5ad943
+REACT_APP_PINAKION_ADDRESS=0x93ed3fbe21207ec2e8f2d3c3de6e058cb73bc04d
+REACT_APP_POLICY_REGISTRY_ADDRESS=0xCf1f07713d5193FaE5c1653C9f61953D048BECe4
+REACT_APP_UNISWAP_V2_ROUTER_02_ADDRESS=0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D
+REACT_APP_UNISWAP_V2_FACTORY_ADDRESS=0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f
+REACT_APP_TOKEN_BRIDGE_ADDRESS=0x88ad09518695c6c3712AC10a214bE5109a655671
+REACT_APP_KLEROS_LIQUID_BLOCK_NUMBER=7303699
+REACT_APP_WEB3_FALLBACK_URL=https://mainnet.infura.io/v3/f1c9535b961648f7bd18209b00e11163
+REACT_APP_APPLICATION_SERVER_KEY=BNqPp3phfZUqu4G1SCkqNuZ_FDL4YuSeAEV97U_G3UbrOuOsTeaI9jHE3QNRJ-O8EH3F2OUdnJJ4ZOhp80FthKU
 
 REACT_APP_KLEROS_LIQUID_ROPSTEN_ADDRESS=0x9643e91d3734b795e914a64169147b70876272ba
 REACT_APP_KLEROS_LIQUID_EXTRA_VIEWS_ROPSTEN_ADDRESS=0x57a3c77595c6A9b6B5A3312c005b0b7B34A19dED
 REACT_APP_PINAKION_ROPSTEN_ADDRESS=0xd042ef7edfa1d39146a25f9cb87b274c422cceb2
 REACT_APP_POLICY_REGISTRY_ROPSTEN_ADDRESS=0x0Ec92A49968eA919dd103117986E9B6076ecAA4A
+REACT_APP_WEB3_FALLBACK_ROPSTEN_URL=https://kovan.infura.io/v3/f1c9535b961648f7bd18209b00e11163
```

Comment:
Hello!
I don't know if this is intentional, but just in case, `REACT_APP_WEB3_FALLBACK_ROPSTEN_URL` is pointing to Kovan instead of Ropsten.

Reply:
LOL. Spotted a good one. Will fix ASAP.
Thanks a lot.

Reply:
No problem!